docs: Quick start guide for up page

spalladino opened the pull request:
Adds a quick start guide for the up page, to be embedded there instead of listed in docs.

Related to #1406

spalladino committed:
Quick start guide for up page

iAmMichaelConnor commented on docs/docs/external/up-quick-start.md:
```diff
@@ -0,0 +1,23 @@
+
+#include_code deploy yarn-project/end-to-end/src/guides/up_quick_start.sh bash
+
+We can check Alice's balance by querying the contract:
```
```suggestion
We can check Alice's private token balance by querying the contract:
```

iAmMichaelConnor commented on docs/docs/external/up-quick-start.md:
```diff
@@ -0,0 +1,23 @@
+
+#include_code declare-accounts yarn-project/end-to-end/src/guides/up_quick_start.sh bash
+
+Start by deploying a private token contract, minting an initial supply to Alice:
```
```suggestion
Start by deploying a private token contract, minting an initial supply of private tokens to Alice:
```

spalladino committed:
Add options to include-code macro

spalladino committed:
Apply suggestions from code review

Co-authored-by: Michael Connor <mike@aztecprotocol.com>

spalladino commented:
Thanks @iAmMichaelConnor, just added the suggestions! I also added some optional tweaks to code blocks, to make it easy to copy them into the static up page. It looks like this in the Docusaurus now:

![Screenshot from 2023-08-15 13-27-57](https://github.com/AztecProtocol/aztec-packages/assets/429604/13b3210b-d460-484b-ba66-f8b16e778821)

iAmMichaelConnor commented on docs/docs/external/up-quick-start.md:
```diff
@@ -0,0 +1,23 @@
+
+#include_code get-balance yarn-project/end-to-end/src/guides/up_quick_start.sh bash
+
+And use Alice's private key to send a transaction to transfer tokens to Bob, and check the result:
```
```suggestion
We can privately transfer tokens to Bob. Only Alice and Bob will know what's happened.

All an external observer will know is that the rules of some smart contract on Aztec have been executed correctly; nothing more.

Here, we use Alice's private key to send a transaction to transfer tokens to Bob, and check the result:
```

iAmMichaelConnor commented on docs/docs/external/up-quick-start.md:
```diff
@@ -0,0 +1,23 @@
+# Quick start
+
+:::info This guide is meant to be included in the up.aztec.network site and not in the main documentation.
```
We'll also need the curl command to install and start the sandbox.

spalladino replied:
Agree, that'll be separate in the up page

spalladino replied:
Copy-paste :-/

iAmMichaelConnor commented on docs/docs/external/up-quick-start.md:
```diff
@@ -0,0 +1,23 @@
+
+:::info This guide is meant to be included in the up.aztec.network site and not in the main documentation.
+
+To interact with the sandbox, install the aztec CLI:
```
Maybe include a link to the longer aztec CLI docs on `aztec CLI`?

iAmMichaelConnor commented on docs/docs/external/up-quick-start.md:
```diff
@@ -0,0 +1,23 @@
+
+And use Alice's private key to send a transaction to transfer tokens to Bob, and check the result:
+
+#include_code transfer yarn-project/end-to-end/src/guides/up_quick_start.sh bash
```
For deeper explanations, and much more information, please visit our docs:
[Insert some links to relevant docs, e.g. the longer getting started guide that lives in the docs, and the docs home page]

spalladino committed:
Add links

spalladino committed:
Link to quick start cli

spalladino committed:
Clarify private transfer
